Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM node:carbon
+
+RUN apt-get update \
+    && apt-get install -y apt-utils \
+    && { \
+        echo debconf debconf/frontend select Noninteractive; \
+        echo mysql-community-server mysql-community-server/data-dir \
+            select ''; \
+        echo mysql-community-server mysql-community-server/root-pass \
+            password '123'; \
+        echo mysql-community-server mysql-community-server/re-root-pass \
+            password '123'; \
+        echo mysql-community-server mysql-community-server/remove-test-db \
+            select true; \
+    } | debconf-set-selections \
+    && apt-get install -y mysql-server
```

Comment:
This is done wrong. Docker containers should be one process. You don't need this.